pkg/nordic_softdevice_ble/src/ble-mac.h

```diff
@@ -0,0 +1,50 @@
+#ifndef BLE_MAC_H
```

Comment:
No doc, no license. Or is this also imported from Nordic? Sorry seems like I was a little too trigger-happy (and something seems to be wrong with Murdock's doc check...).